Gopkg.lock

```diff
@@ -571,30 +571,44 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:199d31b7abebc4e68404defc9e46454a2d73be76db0fd83b553e181d12acc4eb"
+  digest = "1:edd136129d8579865a7af2fc91cdfd2ba6c3ed24b9949b9af0fe9f87b7b3c5f2"
   name = "github.com/portworx/kvdb"
   packages = [
     ".",
     "common",
     "mem",
   ]
   pruneopts = "UT"
-  revision = "3d284390abcf8773537e78f5fb723bad481878ec"
+  revision = "cccaa09abfc969969eff7b4716c42d820b63c4ec"
 
 [[projects]]
-  branch = "kubernetes-1.11"
-  digest = "1:221f5df05abc142715c5a440ec1be954b0d8bd5be46b2684aa8d942fb3addc7d"
+  branch = "master"
+  digest = "1:6ca363d6e0260e1d7edd8c1684d77cfc1611e2b3ea49a5919ebdadbb5c1b6c8a"
   name = "github.com/portworx/sched-ops"
   packages = [
     "k8s",
     "task",
   ]
   pruneopts = "UT"
-  revision = "51b66a2b633767dac6b89254a6dd745e2f3390af"
+  revision = "d64ebc777d0c67885c842bc7f03a4c131cf38bfa"
 
 [[projects]]
   branch = "master"
-  digest = "1:cdff4b38209b651b78fb982e5cbd9ca0520ea41e6f3464139ae6a8876ffdfa05"
+  digest = "1:cfe8d419eb553560ba1e297dc0430a161be3239372a9fd60111e63da3b75fdb8"
+  name = "github.com/portworx/talisman"
```

Comment:
Why is this dependency required?

Reply:
sched-ops master has this in it's vendor. So dep pulled it here too